app/controllers/pipelines_controller.rb

```diff
@@ -66,6 +66,6 @@ def assign_sort_by
   end
 
   def pipeline_params
-    params.require(:pipeline).permit(:name, :description)
+    merge_last_edited_by(params.require(:pipeline).permit(:name, :description))
```

Comment:
This could just be me, and it is pretty minor, so feel free to disregard if you don't agree, but having it all in one statement makes it a bit harder to reason about.

I think it would be clearer if you did:
```
safe_params = ....
merge_last_edited_by(safe_params)
```

Reply:
I don't have a big opinion, I just went your way which seems to make things clearer

Reply:
Cool!